go/mcap/reader.go

```diff
@@ -184,6 +184,20 @@ func (r *Reader) Info() (*Info, error) {
 	return info, nil
 }
 
+// GetAttachment returns an attachment reader located at the specific offset.
```

Comment:
changing `go/mcap` means we should update the library version number in `version.go`

Reply:
thanks, done